.github/workflows/semgrep.yml

```diff
@@ -11,6 +11,7 @@ jobs:
   semgrep:
     name: semgrep/ci
     runs-on: ubuntu-latest
+    if: github.repository == 'cloudflare/circl'
```

Comment:
this seems like it should be the default @hrushikeshdeshpande 

possibly along these lines

```
if: github.repository_owner == 'cloudflare'
```